models.py

```diff
@@ -78,7 +79,11 @@ class Identity(db.Model):
     __tablename__ = 'identity'
     identity = db.Column(db.String(42), primary_key=True)
     created_at = db.Column(db.DateTime)
+    signature = db.Column(db.Text)
+    signed_body = db.Column(db.Text)
```

Comment:
Please add database migrations for these new fields - this will allow us to upgrade dev/prod server databases.
`Flask-Migrate` is already working, `flask db migrate` should create migration automatically, just make sure to check them manually :)

Reply:
decided not to change models at this branch